src/styles/components/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  font-family: ' Roboto ', sans-serif;
```

Comment:
There seems to be extra spacing in the font-family declaration. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be no leading or trailing spaces inside the quotes for font names.